Property_map/include/CGAL/property_map.h

```diff
@@ -274,26 +274,6 @@ Nth_of_tuple_property_map<N, Tuple>
   return Nth_of_tuple_property_map<N, Tuple>();
 }
 
-/// \ingroup PkgProperty_map
-/// Struct that turns a property map into a unary functor with
-/// `operator()(key k)` calling the get function with `k`
-template <class PropertyMap>
-struct Property_map_to_unary_function{
```

Comment:
1. `Property_map_to_unary_function` is in CGAL-4.8 and documented. You cannot remove it that way.
2. `boost::property_map_function` does not seem documented.

I tend to object to that change, unless you have very good reasons.